one/commands/auth.py

```diff
@@ -74,6 +74,29 @@ def azure():
     shutil.move(CLI_ROOT + '/.env', CLI_ROOT + '/secrets')
 
 
+@auth.command(help='Authentication using Okta SSO provider.')
+def okta(auth_image=None):
```

Comment:
Hi @sohflp, in this case, there is no need to pass `auth_image=None`.

Reply:
Hey @arthurbdiniz, I removed the parameter as suggested.